androidHyperskillApp/src/main/java/org/hyperskill/app/android/step/view/delegate/StepDelegate.kt

```diff
@@ -121,4 +133,36 @@ class StepDelegate<TFragment>(
                 .checkNotificationChannelAvailability(context, HyperskillNotificationChannel.DailyReminder)
         }
     }
+
+    @DrawableRes
+    private fun getShareStreakDrawableRes(streak: Int): Int =
```

Comment:
`ShareStreakDialogFragment` also has the same function `getImageRes`, maybe it's better to provide image resource for `ShareStreakDialogFragment` by argument